api/v1beta1/vmagent_types.go

```diff
@@ -202,6 +202,11 @@ type VMAgentSpec struct {
 	// +optional
 	ProbeNamespaceSelector *metav1.LabelSelector `json:"probeNamespaceSelector,omitempty"`
 
+	// NodeScrapeSelector defines VMNodeScrape to be selected for scraping.
+	// if neither PodScrapeNamespaceSelector, nor ProbeSelector nor PodScrapeSelector, nor NodeScrapeSelector are specified,
```

Comment:
This line mentions NodesScrapeSelector but similar lines above are not updated to follow this line.